File: app/src/main/java/ru/roscha_akademii/medialib/videocontrol/VideoControlInterface.java
package ru.roscha_akademii.medialib.videocontrol;

import com.google.android.exoplayer2.ExoPlayer;

import ru.roscha_akademii.medialib.videocontrol.view.VideoControlView;

public interface VideoControlInterface {
    void setPlayer(ExoPlayer player);

    void releasePlayer();

    void setCallback(VideoControlCallback callback);

    void setFullscreenAction(VideoControlView.FullscreenMode mode);
}

File: app/src/main/java/ru/roscha_akademii/medialib/videocontrol/VideoControlInterface.kt
package ru.roscha_akademii.medialib.videocontrol

import com.google.android.exoplayer2.ExoPlayer

import ru.roscha_akademii.medialib.videocontrol.view.VideoControlView

interface VideoControlInterface {
    fun setPlayer(player: ExoPlayer)

    fun releasePlayer()

    fun setCallback(callback: VideoControlCallback)

    fun setFullscreenAction(mode: VideoControlView.FullscreenMode)
}
